routes/courses.js

```diff
@@ -24,17 +24,17 @@ router.get('/new', function (req, res) {
 });
 
 router.post('/', (req, res) => {
-    UserModel.findById(req.body.userId)
+    UserModel.findById(req.body.userId).exec()
```

Comment:
I think `exec()` is used when you need a fully featured promise, but we only need `.then()` here, which should be supported without `.exec()`